src/communication/xcConnection.ts

```diff
@@ -29,8 +31,10 @@ export class DefaultConnection implements Connection {
             });
         };
         let errorListener = (err: Error) => {
-            console.error("getModel request failed");
-            console.error(err);
+            if (isDebugEnabled()) {
```

Comment:
this is not needed. log debug doesn't do anything when debug id not enabled.
testing isDebugEnabled is only necessary when you need to construct dynamic strings before calling the logger... and this is not the case here.